core/src/avm2/globals.rs

```diff
@@ -698,7 +698,7 @@ fn load_playerglobal<'gc>(
                 // Lookup with the highest version, so we we see all defined classes here
                 let ns = Namespace::package($package, ApiVersion::VM_INTERNAL, &mut activation.borrow_gc());
                 let name = QName::new(ns, $class_name);
-                let class_object = activation.domain().get_defined_value(activation, name)?;
+                let class_object = activation.domain().get_defined_value(activation, name).unwrap_or_else(|e| panic!("Failed to lookup {name:?}: {e:?}"));
```

Comment:
Is this change leftover from debugging or generally useful?

Reply:
This gives a much more useful error - our normal error-printing logic doesn't really work during playerglobal loading, since the `Error` class might not have been initialized.